reviews/2021-cloud-custodian-annual.md

```diff
@@ -0,0 +1,199 @@
+    
+- [State of Cloud Custodian 2021](https://youtu.be/DB52-yaIrqY)
+  - A summary video for our community, covers many aspects of the project
+
+## Goals for 2022
+
+- New enhancement process for feature development
+- GCP moving from beta to GA
+- ARM64 docker images
+- Extracting core from the AWS provider (have each cloud provider in its own package)
+- Kubernetes Admission Controller
+- Policy Enforcement against IaC definitions like Terraform (aka shift left)
+- End User Policy Testing framework
+
+# Incubation Requirements
+
+## Dev Statistics
+"Have a healthy number of committers"
+"Demonstrate a substantial ongoing flow of commits and merged contributions"
+
+- [c7n devstats](https://cloudcustodian.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m&viewPanel=4)
+
+| Statistic | Sandbox | Current | Growth | 
+|-|-|-|-|
+| GitHub Stars | 2,940+ | 3,888 | 32% |
+| Releases     | 77+    | 116+  | 50% | 
+| Commits      | 3,292+ | 3,677 | 12% |
+| Forks        | 856+   | 1,100+| 29% |
+| Contributors | 258+   | 320+  | 24% |
+| Maintainers  | 14     | 10    | -29% |
+| Affiliations between Maintainers | 4 | 6 | 50% |
+
+## Maintainers
+
+- [Current Maintainers](https://github.com/cloud-custodian/cloud-custodian/blob/master/OWNERS.md)
+
+#### Leads
+
+- Kapil Thangavelu @kapilt (@kapilt) - Project Lead - Stacklet
+
+#### Aws Provider Maintainers
+
+- Todd Stansell (@tjstansell) - 23andme
+- John Hillegass (@JohnHillegass) - Fidelity
+- Sonny Shi (@thisisshi) - Stacklet
+- David Filiatrault (@FireballDWF) - Amazon
+- Jamison Roberts (@jtroberts83) - Stacklet
+
+#### Gcp Provider Maintainers
+
+- Karol Lassak (@ingwarsw) - Ingwar & co.
+
+#### Azure Provider Maintainers
+
+- Stefan Gordon (@stefangordon) - Stacklet
+- Andy Luong (@aluong) - Microsoft
+- Kiril Logachev (@logachev) - Stacklet
+
+#### Emeritus
+
+Since the opensource inception of Cloud Custodian in April 2016, we have had numerous maintainers who over the course of years are no longer current contributors, but we still celebrate their contributions and they are listed here.
+
+- Kit Ewbank (@ewbankkit)
+- Mandeep Bal (@mandeepbal)
+- Erin Welch (@erwelch)
+- Darcy Laylock (@Sutto)
+
+# Security
+"Clearly documented security processes explaining how to report security issues to the project"
+
+Dedicated [security mailing list](https://github.com/cloud-custodian/cloud-custodian#security) for potential vulnerability reports. 
+
+More security details in: 
+[Joint Review](https://github.com/cncf/tag-security/blob/cf1f38ff803fb3f830209e16618221844a9dea68/assessments/projects/custodian/joint-review.md)
+[Self-assessment](https://github.com/cncf/tag-security/blob/ef2d51d54697f4037a4a697b405d270df2579620/assessments/projects/custodian/self-assessment.md)
+
+Note: Both documents pending merge, will update accordingly
+
+# Adoption
+
+[ADOPTERS.md](https://github.com/cloud-custodian/cloud-custodian/ADOPTERS.md)
```

Comment:
404

Reply:
@castrojo - where should this link to?

Reply:
Fixed, sorry for the late response, I was at a conference!